### ISSUE TITLE
MeanCloud__ForgesHold1M: Mulitple issues with smelting and game hang

### ISSUE THREAD

**Issue by doughphunghus** [doughphunghus/MeanCloud_7DtD#11]
From: https://community.7daystodie.com/topic/17197-doughs-modlets/page/9/?tab=comments#comment-423073
game version: unknown

Issue:
"...can't seem to access my forges while they are still smelting my resources like iron. I can access them after they are done and once they are done  smelting say Iron I can then access my forges. .... I just put clay into my forges and my console went mental with red error messages and I couldn't move or even cancel the console log, I had to Alt+F4 out of the game. "

**Comment by doughphunghus** [comment]
EDIT: I just tested MeanCloud__ForgesHold1M mod on a19.4 b7 (as the only mod loaded) and was unable to reproduce this issue. Forge accessible while smelting iron/clay/etc, no console errors, etc.  I did not try to smelt 1,000,000 items with a secondary mod, just noting "basic mod" appears functional.  The last major issue with this mod was because another person loaded another forge mod and they did not play together.

Leave open, as user is being asked to provide more info.
